composer.json

```diff
@@ -16,11 +16,12 @@
         "symfony/framework-bundle": "^2.8|^3.0",
```

Comment:
should we set this to ^2.8|^3.3 ?

Reply:
this would drop 3.0, 3.1, 3.2 then?

Reply:
yes. those are not maintained anymore. i think with every minor version change, we should drop all unmaintained symfony versions to keep the test matrix reasonable and avoid issues.

Reply:
@dbu how long shall we support Symfony 2.8?

Reply:
started wondering that as well, what with symfony 4 coming up. i'd say if its no extra effort, we can keep allowing ^2.8.18 (has the new phpunit version with namespaces). then we can drop 2.8 with the next minor version. if its an issue to support both, i'd drop it now.